pkg/apis/flowcontrol/types.go

```diff
@@ -223,17 +234,25 @@ type PriorityLevelConfigurationSpec struct {
 	// `queues` is a number of queues that belong to a non-exempt PriorityLevelConfiguration object. The queues exist
 	// independently at each apiserver. The value must be positive for a non-exempt priority level and setting it to 1
 	// disables shufflesharding and makes the distinguisher method irrelevant.
+	// TODO: sugguest a default or a way of deciding on a value.
 	// +optional
 	Queues int32
 	// `handSize` is a small positive number for applying shuffle sharding. When a request arrives at an apiserver the
 	// request flow identifier’s string pair is hashed and the hash value is used to shuffle the queue indices and deal
 	// a hand of the size specified here. If empty, the hand size will the be set to 1.
+	// NOTE: To figure out a better value for your cluster, please refer to (#76846)[https://github.com/kubernetes/kubernetes/issues/76846#issuecomment-523700960]
 	// +optional
 	HandSize int32
 	// `queueLengthLimit` is a length limit applied to each queue belongs to the priority.  The value must be positive
 	// for a non-exempt priority level.
 	// +optional
 	QueueLengthLimit int32
+	// `exempt` defines whether the priority level is exempted or not.  There should be at most one exempt priority level.
```

Comment:
I do not think you intend all the statements in this comment block.  Can, and probably should, be fixed in follow-up.